bindings-graph/bindings-graph.mjs

```diff
@@ -70,7 +65,19 @@ class CanjsDevtoolsBindingsGraph extends StacheElement {
 						case "success":
 							vm.bindingsError = null;
 							vm.selectedObj = detail.selectedObj;
-							vm.availableKeys.replace(detail.availableKeys);
+
+							// ObservableArray.splice dispatches length event no matter what
```

Comment:
Can you explain this? Would it make sense to change the `<bindings-graph>` component to not change `selectedKey` when the `availableKeys` length changes if `selectedKey` is already set?